assets/js/components/indicators.js

```diff
@@ -298,12 +298,18 @@ export class IndicatorSection {
     this.comparisonButtonsContainer.insertBefore(this.chartContainer);
   }
 
+  highlightComparisonButton(section, muniId) {
+    $(section + ' .chart-btn').css('border', 'none');
+    $(section + ' #' + muniId).css('border', '1px solid black');
+  }
+
   _updateComparisonButtons() {
     this.comparisonButtonsContainer.empty();
     this.comparisonButtonsContainer.append("Comparing ");
     [this.chartData(), ...(this.comparisons)].forEach((comparison) => {
-      const button = $(' <button class="button" style="display: unset; margin: 2px 2px"></button> ');
+      const button = $(' <button id="' + comparison.municipality.code + '"class="button chart-btn" style="display: unset; margin: 2px 2px"></button> ');
```

Comment:
Likewise an initial class for the button rather than an inline style.

Reply:
Changed

Reply:
Can the `display: unset; margin: 2px 2px` inline style be part of the `.chart-btn`css class rather?

Reply:
Ah yes, changed